deployments/get/objects.go

```diff
@@ -550,7 +552,7 @@ func GenerateDaemonSet(env, namespace string) *appsv1.DaemonSet {
 							Name:  "init",
 							Image: "kubearmor/kubearmor-init:latest",
 							SecurityContext: &corev1.SecurityContext{
-								Privileged: &privileged,
+								Privileged: &kubearmorInitPrivileged,
```

Comment:
Is this necessary? Can't we bypass this with some specific capability add?

Reply:
Yeah normally `SYS_ADMIN` should suffice for mounting filesystems but bpffs is an exception ig. (based on my tries with both privileged true and false, couldn't find a source stating this yet)

Reply:
I don't think we will go ahead with this, if it requires privileged flag.

Reply:
Cilium also doesn't depend on it, so can you please verify things again? Thanks.

Reply:
I'll look into Cilium as well. Calico seemed to be doing the same - https://github.com/projectcalico/calico/blob/a8bdcd60f99f1ab3b0495b7b4fe4d1d3fb719c35/manifests/calico.yaml#L4526-L4547 :thinking: